acquire/diagnostics/linux.py

```diff
@@ -0,0 +1,232 @@
+import json
+import os
+import platform
+import stat
+import subprocess
+from pathlib import Path
+
+
+def check_sudo() -> bool:
+    return os.geteuid() == 0
+
+
+def get_disk_info() -> dict:
```

Comment:
Adding suggestions for this all in GitHub is a bit slow, but can you add some newlines between sensible sections/parts?